blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --v2-feature-carousel-cards-gutter: 16px;
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-width: 48px;
+  --v2-feature-carousel-icon-height: 48px;
+  --v2-feature-carousel-svg-width: 24px;
+  --v2-feature-carousel-svg-height: 24px;
+}
+
+.v2-feature-carousel-container {
+  background-color: var(--c-grey-50);
+}
+
+.v2-feature-carousel-container .v2-feature-carousel-wrapper {
+  max-width: 1390px;
```

Comment:
Could we use --wrapper-width?

Reply:
Used this to match design , however removed this as be default the div's ave wrapper-width